app.js

```diff
@@ -1,16 +1,139 @@
 /* Imports */
+import { getRandomItem } from './utils.js';
 
 /* State */
+let gameState = ''; // 'guess' or 'results'
+let userPlay = ''; //'rock' or 'paper' or 'scissors'
+let computerPlay = ''; //'rock' or 'paper' or 'scissors'
+let result = ''; // 'win' or 'lose' or 'draw'
+
+let wins = 0;
+let losses = 0;
+let draws = 0;
+let totalPlays = 0;
+
+// probability array
+const choices = ['rock', 'paper', 'scissors'];
 
 /* Actions */
-function load() {}
+function loadPage() {
+    displayGuess();
+    displayResults();
```

Comment:
Your scoreboard is updating when you click the play again button instead of right when they win because you are calling displayResults() here in the loadPage function that is only called when you click play again. If you want the scoreboard to update when you see the 'You win' result for example you will need to move this into the play game function. We can go over this later if you would like!

Reply:
Thanks, Aidan, that's really helpful!